core/blockchain.go

```diff
@@ -1550,6 +1552,9 @@ func (bc *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 	if len(rebirthLogs) > 0 {
 		bc.logsFeed.Send(mergeLogs(rebirthLogs, false))
 	}
+	if len(deletedLogs) > 0 || len(rebirthLogs) > 0 {
+		bc.hubbleFeed.Send(append(mergeLogs(deletedLogs, false), mergeLogs(rebirthLogs, false)...))
```

Comment:
I think we can use mergeLogs(deletedLogs, true) and this will sort the logs descending block number.Thus we wont have to sort in contracts_events_processor.  
Though mergeLogs is sorting by blockNumber but not sorting logs in a block by index. 
Please verify if we can use this.

Reply:
`mergeLogs : true` will sort in ascending order by blockNumber. I do believe that we do not need another sorting logic in events_processor, but I kept it anyway because there was no easy way to test.